app/src/main/res/layout/activity_location_picker.xml

```diff
@@ -32,7 +32,7 @@
             android:id="@+id/map"
             android:name="com.google.android.gms.maps.SupportMapFragment"
             android:layout_width="match_parent"
-            android:layout_height="700dp"
+            android:layout_height="725dp"
```

Comment:
This doesn't seem to fix the bar going too low at the bottom and resulting in:
``` 
20 K
M
S
```

Reply:
That's the Map Fragment.

Reply:
Looks about alright